Sources/_FoundationCShims/include/_CStdlib.h

```diff
@@ -156,6 +156,10 @@
 #ifndef TZDEFAULT
 #define TZDEFAULT    "/etc/localtime"
 #endif /* !defined TZDEFAULT */
+#elif TARGET_OS_WINDOWS
+/* not required */
+#else
+#error "possibly define TZDIR and TZDEFAULT for this platform"
```

Comment:
Are we going to cause a new error on any other platforms with this?

Reply:
Prior to this change, if there are other platforms that require TZDIR/TZDEFAULT that aren't TARGET_OS_MAC, TARGET_OS_LINUX, TARGET_OS_BSD or TARGET_OS_WINDOWS, this manifests as a missing symbol error where they are referenced, so there should be no _new_ errors.

Reply:
Sounds good